react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicked: 0, color: 'white' };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    const newClickNum = this.state.clicked + 1;
+
+    this.setState({ clicked: newClickNum });
+
+    const colorList = ['darkblue', 'purple', 'hotpink', 'orange', 'yellow', 'white'];
+    let newColor = '';
+    if (Number.isInteger((newClickNum) / 3) && (newClickNum) < 19) {
+      newColor = colorList[(newClickNum) / 3 - 1];
+      this.setState({ color: newColor });
+    }
```

Comment:
Very clever solution to get the class name! After removing `color` from state you will need to refactor this a bit. You should move this into its own method named something like `getClass`, the method should return the correct class based on the `clicked` property in state.